requirements/typecheck.in

```diff
@@ -1 +1,6 @@
+-r prod.txt
```

Comment:
Should this be part of the template?

Reply:
Yeah, the `requirements/*.in` files are generally templated. `make typecheck` should run successfully on a fresh new project created by the cookiecutter, so the requirements file will be needed.